koku/api/organizations/queries.py

```diff
@@ -131,6 +131,13 @@ def _get_filter(self, delta=False):  # noqa: C901
                 for item in filter_value:
                     q_filter = QueryFilter(parameter=item, **filter_obj)
                     filters.add(q_filter)
+            access = self.parameters.get_access(filter_key)
+            filt = self.FILTER_MAP.get(filter_key, [])
+            if access and filt:
+                filt = self.FILTER_MAP.get(filter_key)
```

Comment:
i don't think you need to redefine `filt` right here since it is defined on line 135.

Reply:
Good call, removed